update.go

```diff
@@ -184,8 +183,15 @@ other options are ignored.
 		config.Cgroups.Resources.KernelMemoryTCP = int64(*r.Memory.KernelTCP)
 		config.Cgroups.Resources.Memory = int64(*r.Memory.Limit)
 		config.Cgroups.Resources.MemoryReservation = int64(*r.Memory.Reservation)
-		config.Cgroups.Resources.MemorySwap = int64(*r.Memory.Swap)
```

Comment:
Since we're using `int64` here anyway, why don't we just switch the parsing stage to `ParseInt` as well? Or make it the backup?